pwem/tests/__init__.py

```diff
@@ -283,3 +283,11 @@ def defineDatasets():
                 'allMics': 'micrographs/*.mrc',
                 'autoPickingCoordinates': 'autoPickingCoordinates'
             })
+
```

Comment:
Datasets can be define inside plugins. This is not the right place for this one. ;-)

Reply:
Hi Pablo, I tried defining it inside the scipion-chem plugin but seems like it didn't work.
I guess the datasets definition goes over some plugins but maybe not over the scipion-chem, that's why I added it here.
Should we change it otherway?